Source/WarpX.H

```diff
@@ -556,6 +558,7 @@ public:
     void setplot_rho (bool a_plot_rho) {plot_rho = a_plot_rho;}
 
     int maxStep () const {return max_step;}
+
```

Comment:
I'd avoid newlines like this that'll put us out of sync with upstream. Same in particle boundary buffer .cpp and _libwarpx.py